Content.Server/Shuttles/Systems/ArrivalsSystem.cs

```diff
@@ -241,6 +257,41 @@ private void OnPlayerSpawn(PlayerSpawningEvent ev)
         }
     }
 
+    private bool TeleportToMapSpawn(EntityUid player, EntityUid stationId, TransformComponent? transform = null)
```

Comment:
probably should be called `TryTeleportToMapSpawn`. That's just a convention for functions that return booleans indicating success/failure.